app/Services/StandService.php

```diff
@@ -475,6 +577,45 @@ public function getAircraftEligibleForArrivalStandAllocation(): Collection
         )->get();
     }
 
+    private function getDepartureStandsToAssign(): Collection
+    {
+        return NetworkAircraft::join('aircraft_stand', 'network_aircraft.callsign', '=', 'aircraft_stand.callsign')
+            ->join('stands', 'aircraft_stand.stand_id', '=', 'stands.id')
+            ->join('airfield', 'airfield.id', '=', 'stands.airfield_id')
+            ->leftJoin('stand_assignments', 'network_aircraft.callsign', '=', 'stand_assignments.callsign')
+            ->whereRaw('aircraft_stand.stand_id <> stand_assignments.stand_id')
+            ->orWhereNull('stand_assignments.stand_id')
+            ->whereRaw('airfield.code = network_aircraft.planned_depairport')
+            ->select(['network_aircraft.*', 'aircraft_stand.stand_id'])
```

Comment:
This wildcard makes me a little uneasy - do you know exactly which columns you need?

Reply:
Essentially we're populating the network_aircraft model, so I guess we want every column on that table?

Reply:
Hmmm, okay that seems reasonable to me